### ISSUE TITLE
When focus comes to editor field for the first time, narrator is not reading the code present at the line

### ISSUE THREAD

**Issue by InduPriya1805** [microsoft/monaco-editor#1752]
**Environment Details:​**
www.typescriptlang.org/​​
​​
**Browser Details: ​​​**
Microsoft Edge 44.18362.1.0​
Screen Reader: Narrator​
​
**Repro Steps:** ​​​
1) Turn on narrator 
2) Hit the URL "www.typescriptlang.org"​​
3) Navigate till "Playground" link and press enter​​
4) Tab till editor field and verify narrator is reading the code at the line or not​

**Actual Result: ​​​**
When focus comes to editor field for the first time, narrator is not reading the code present at the line​
​​
**Expected Result: ​​​**
When focus comes to editor field for the first time, narrator should also read the code present at the line along with the instructions​
**​
User Impact: ​​​**
If narrator does not read anything then screen reader users might think there is no code given at the line​

**Attachment:**
[Bug_1752_narrator is not reading code at 1st line in editor.pptx](https://github.com/microsoft/monaco-editor/files/4021386/Bug_1752_narrator.is.not.reading.code.at.1st.line.in.editor.pptx)

**Comment by alexdima** [comment]
I believe this might be a duplicate of #478

**Comment by alexdima** [comment]
Closing, as Microsoft Edge Legacy has reached end of support on March 9, 2021. See [the end of support for Microsoft Edge Legacy announcement](https://docs.microsoft.com/en-us/lifecycle/announcements/edge-legacy-eos-details)
